Tests/SafeDIMacrosTests/InstantiableMacroTests.swift

```diff
@@ -1258,6 +1258,47 @@ final class InstantiableMacroTests: XCTestCase {
         }
     }
 
+    func test_declaration_fixit_addsFixitMissingRequiredInitializerWhenPropertyIsMissingInitializerAndThereAreNoDependencies() {
+        assertMacro {
+            """
+            @Instantiable
+            public struct ExampleService {
+                let uninitializedProperty: Int
+            }
+            """
+        } diagnostics: {
+            """
+            @Instantiable
+            public struct ExampleService {
+                                         ╰─ 🛑 @Instantiable-decorated type with no @Instantiated, @Received, or @Forwarded-decorated properties and at least one uninitialized property must have an empty `public` or `open` initializer
```

Comment:
@tareksabry1337 I think this should help resolve the issue you encountered in #53